doc/private_net.rst

```diff
@@ -0,0 +1,101 @@
+::
+  {
+    "sealEngine": "Ethash",
+    "params": {
+      "accountStartNonce": "0x00",
+      "maximumExtraDataSize": "0x20",
+      "homesteadForkBlock": "0x00",
+      "daoHardforkBlock": "0x00",
+      "EIP150ForkBlock": "0x00",
+      "EIP158ForkBlock": "0x00",
+      "byzantiumForkBlock": "0x00",
+      "constantinopleForkBlock": "0x00",
+      "constantinopleFixForkBlock": "0x00",
+      "minGasLimit": "0x5208",
+      "maxGasLimit": "0x7fffffffffffffff",
+      "tieBreakingGas": false,
+      "gasLimitBoundDivisor": "0x0400",
+      "minimumDifficulty": "0x100000",
+      "difficultyBoundDivisor": "0x0800",
+      "durationLimit": "0x0d",
+      "blockReward": "0x4563918244F40000",
+      "networkID" : "0x42",
+      "chainID": "0x42",
+      "allowFutureBlocks" : false
+    },
+    "genesis": {
+      "nonce": "0x0000000000000042",
+      "difficulty": "0x100000",
+      "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+      "author": "0x0000000000000000000000000000000000000000",
+      "timestamp": "0x5A5B92D7",
+      "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+      "extraData": "0x655741534d2074657374206e6574776f726b2030",
+      "gasLimit": "0x989680"
+    },
+    "accounts": {
+      "0000000000000000000000000000000000000001": { "precompiled": { "name": "ecrecover", "linear": { "base": 3000, "word": 0 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000002": { "precompiled": { "name": "sha256", "linear": { "base": 60, "word": 12 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000003": { "precompiled": { "name": "ripemd160", "linear": { "base": 600, "word": 120 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000004": { "precompiled": { "name": "identity", "linear": { "base": 15, "word": 3 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000005": { "precompiled": { "name": "modexp" }, "balance": "0x01" },
+      "0000000000000000000000000000000000000006": { "precompiled": { "name": "alt_bn128_G1_add", "linear": { "base": 500, "word": 0 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000007": { "precompiled": { "name": "alt_bn128_G1_mul", "linear": { "base": 40000, "word": 0 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000008": { "precompiled": { "name": "alt_bn128_pairing_product" }, "balance": "0x01" },
+      "008a78302c6fe24cc74008c7bdae27b7243a7066" /* <= Enter your account address here */: {
+          "balance" : "0x200000000000000000000000000000000000000000000000000000000000000"
+      }
+    }
+  }
+  
+  
+This includes all forks activated at genesis, all precompiled contracts existing on the main net and initial balance for your account.
+
+In case this format gets out of date, please consult the format of the source files in https://github.com/ethereum/aleth/tree/master/libethashseal/genesis.
```

Comment:
Can rephrase as something like: "Note that this format can change occasionally. For the latest format, please consult the source files in"